Userland/Libraries/LibX86/Types/u256.h

```diff
@@ -147,6 +147,11 @@ class u256 {
     }
 
     // bitwise
+    ALWAYS_INLINE constexpr u128 operator~() const
```

Comment:
This should probably return a `u256` :^)

Reply:
That is indeed the case, will be fixed when I get to coding next